code/drasil-printers/lib/Language/Drasil/Printing/Import/Symbol.hs

```diff
@@ -17,13 +17,18 @@ symbol (Integ    n) = P.Int (toInteger n)
 symbol (Special  s) = P.Spec s
 --symbol (Greek g)    = P.Gr g
 symbol (Concat  sl) = P.Row $ map symbol sl
---
--- Handle the special cases first, then general case
-symbol (Corners [] [] [x] [] s) = P.Row [P.Row [symbol s, P.Sup $ symbol x]]
-symbol (Corners [] [] [] [x] s) = P.Row [P.Row [symbol s, P.Sub $ symbol x]]
-symbol (Corners [_] [] [] [] _) = error "rendering of ul prescript"
-symbol (Corners [] [_] [] [] _) = error "rendering of ll prescript"
-symbol Corners{}                = error "rendering of Corners (general)"
+symbol (Corners ul dl ur dr s)  =
+  let
+    mergeSymbols :: [Symbol] -> P.Expr
+    mergeSymbols [s'] = symbol s'
+    mergeSymbols ss   = P.Row $ map symbol ss 
```

Comment:
I'm guessing this flattening (instead of a nested `P.Row`) is what causes a level of `{}` to go away. I'm pretty sure the nested `P.Row` was inserted explicitly for the latex to have more `{}` in it at some point!!!!!

Reply:
I think this was just incomplete. Searching through the issue tracker leads to #1606.

Reply:
Lo and behold, that exact issue happens: <img width="364" alt="Screenshot 2025-04-01 at 3 33 04 PM" src="https://github.com/user-attachments/assets/8570d04f-05b1-4235-8101-77079890bc53" />

Reply:
Ah ah! So we do need to take some care.